examples/notebooks/beam-ml/run_inference_tensorflow_hub.ipynb

```diff
@@ -0,0 +1,307 @@
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "b22e51ad-cbf2-4dfe-a1c6-3ad17d5a3749",
+        "scrolled": true
+      },
+      "outputs": [],
+      "source": [
+        "!pip install apache_beam[gcp]==2.41.0\n",
+        "!pip install apache-beam[interactive]==2.41.0\n",
+        "!pip install tensorflow==2.10.0\n",
+        "!pip install tensorflow_text==2.10.0\n",
+        "!pip install keras==2.10.0\n",
+        "!pip install tfx_bsl==1.10.0\n",
+        "!pip install pillow==8.4.0"
+      ],
+      "id": "b22e51ad-cbf2-4dfe-a1c6-3ad17d5a3749"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "f313a508-59ea-47ed-86eb-c9c8e67785f2",
+        "scrolled": true
+      },
+      "outputs": [],
+      "source": [
+        "import os\n",
+        "import importlib\n",
+        "\n",
+        "import numpy as np\n",
+        "import tensorflow as tf\n",
+        "import tensorflow_hub as hub\n",
+        "import tensorflow_text\n",
+        "\n",
+        "from tensorflow import keras\n",
+        "\n",
+        "from typing import Any\n",
+        "from typing import Dict\n",
+        "from typing import Iterable\n",
+        "from typing import Optional\n",
+        "from typing import Sequence\n",
+        "\n",
+        "import apache_beam as beam\n",
+        "import apache_beam.runners.interactive.interactive_beam as ib\n",
+        "\n",
+        "from apache_beam.ml.inference.base import RunInference\n",
+        "from apache_beam.ml.inference.base import ModelHandler\n",
+        "from apache_beam.ml.inference.base import PredictionResult\n",
+        "from apache_beam.internal import pickler\n",
+        "from apache_beam.runners.runner import PipelineResult\n",
+        "from apache_beam.runners.interactive.interactive_runner import InteractiveRunner\n",
+        "\n",
+        "from tfx_bsl.public.beam.run_inference import CreateModelHandler\n",
+        "from tfx_bsl.public.proto import model_spec_pb2"
+      ],
+      "id": "f313a508-59ea-47ed-86eb-c9c8e67785f2"
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "74db0203-3d26-4bc4-8271-81fad9756297"
+      },
+      "source": [
+        "## Create a Keras Model from TensorFlow Hub Image"
+      ],
+      "id": "74db0203-3d26-4bc4-8271-81fad9756297"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "r1fgn5I_-mzA"
+      },
+      "outputs": [],
+      "source": [
+        "from google.colab import auth\n",
+        "auth.authenticate_user()"
+      ],
+      "id": "r1fgn5I_-mzA"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "2ff8e394-f577-4dea-bef9-a4f4528c1378"
+      },
+      "outputs": [],
+      "source": [
+        "PROJECT_ID = '<Project Id>'\n",
+        "GCS_BUCKET = '<GCS Bucket>'\n",
```

Comment:
Also, can you provide instructions for the users on how they need to use the `GCS_BUCKET`? 

for example: Download the model from `some_link` and place it in the `GCS_BUCKET`

Reply:
I think the model eventually gets saved in the GCS_BUCKET, so it should just be an empty bucket. I agree that adding an explanatory comment would be helpful though, and maybe changing `<GCS BUCKET>` to `gs://<GCS_BUCKET>` so that the desired formatting is clear

Reply:
Changed `MODEL_PATH` to `MODEL_EXPORT_DIR = f'gs://{GCS_BUCKET}/st5-base/1'`. Is this the intended format? @damccorm

Reply:
That works, could you add a comment explaining what this is though?

Reply:
Added clarifications for this.